mappings/mp_efo_impc.sssom.tsv

```diff
@@ -0,0 +1,100 @@
+# mapping_provider: https://www.mousephenotype.org
+# mapping_set_description: 'Some MP-EFO mappings curated by Violeta Munoz-Fuentes for IMPC.'
+# mapping_set_id: https://w3id.org/sssom/commons/mouse-human/mappings/mp_efo_impc.sssom.tsv
+subject_id	subject_label	predicate_id	object_id	object_label	mapping_justification	subject_source	object_source	confidence	mapping_date	author_id
+MP:0000013	abnormal adipose tissue distribution	skos:closeMatch	EFO:0007800	body fat percentage	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000180	abnormal circulating cholesterol level	skos:narrowMatch	EFO:0004574	total cholesterol measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000180	abnormal circulating cholesterol level	skos:narrowMatch	EFO:0004611	low density lipoprotein cholesterol measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000180	abnormal circulating cholesterol level	skos:narrowMatch	EFO:0008592	high density lipoprotein particle size measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000181	abnormal circulating LDL cholesterol level	skos:exactMatch	EFO:0004611	low density lipoprotein cholesterol measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000184	abnormal circulating HDL cholesterol level	skos:exactMatch	EFO:0008592	high density lipoprotein particle size measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000188	abnormal circulating glucose level	skos:exactMatch	EFO:0004468	glucose measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000199	abnormal circulating serum albumin level	skos:exactMatch	EFO:0004535	serum albumin measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000202	abnormal circulating alkaline phosphatase level	skos:exactMatch	EFO:0004533	alkaline phosphatase measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000203	abnormal circulating aspartate transaminase level	skos:exactMatch	EFO:0004736	aspartate aminotransferase measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000226	abnormal mean corpuscular volume	skos:exactMatch	EFO:0004526	mean corpuscular volume	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0001259	abnormal body weight	skos:exactMatch	EFO:0004338	body weight	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0001553	abnormal circulating free fatty acids level	skos:broadMatch	EFO:0005110	fatty acid measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0001560	abnormal circulating insulin level	skos:broadMatch	EFO:0004467	insulin measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140

```

Comment:
This is inconsistent with the `skos:exactmatch`case above, but it looks correct.